ProcessMaker/processmaker
# FOUR-6307 DevDependencies that should be Dependencies and vice-versa

## Issue & Reproduction Steps
https://processmaker.atlassian.net/browse/FOUR-6307
* Also. Removed the `coverage` folder since it was being committed for some reason while testing jest but not fixing jest in this ticket

## Solution
- Moved the dependencies to the corrective places and removed some dependencies that needed be to be listed

## How to Test
Just make sure you can install the dependencies without any errors and compile the app without any issues and that the app runs without any issues

## Related Tickets & Packages
- Link to any related FOUR tickets, PRDs, or packages

## Code Review Checklist
